examples/my_kentik_portal_example_test.go

```diff
@@ -74,18 +74,18 @@ func getAllTenants() error {
 func pickTenantID() (models.ID, error) {
```

Comment:
I think it would be better to return Optional ID as this function is not guaranteed to find a tenant

Reply:
IMO it would unnecessarily complicate the function. It always returns valid tenant ID unless there is an error. User should not use returned value if there is an error.

More of a problem is that function panics if there are 0 tenants.